### PR TITLE
[Merged by Bors] - feat: add 3 lemmas linking iSup and symmDiff

### PR DESCRIPTION
**Pull request by EtienneC30**
Add 3 lemmas to show that the symmetric difference of two `iSup`s/`biSup`s is less or equal to the `iSup`/`biSup` of the symmetric differences.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/SymmDiff.lean [hunk 1]
```diff
@@ -215,6 +215,11 @@ theorem symmDiff_triangle : a ∆ c ≤ a ∆ b ⊔ b ∆ c := by
   rw [sup_comm (c \ b), sup_sup_sup_comm, symmDiff, symmDiff]
 #align symm_diff_triangle symmDiff_triangle
 
+lemma le_symmDiff_sup {α : Type*} [GeneralizedCoheytingAlgebra α] (x y : α) : x ≤ (x ∆ y) ⊔ y := by
+  rw (config := {occs := .pos [1]}) [← symmDiff_bot x]
+  rw (config := {occs := .pos [2]}) [← symmDiff_bot y]
+  exact symmDiff_triangle ..
```
Comment (fpvandoorn) [inline Mathlib/Order/SymmDiff.lean:221]:
```suggestion
  convert symmDiff_triangle x y ⊥ <;> rw [symmDiff_bot]
```
if you want, this is a shortened proof. But feel free to leave it as-is.

**Review by fpvandoorn — COMMENTED** [review]
Thanks for your contribution. Looks good, but I have a few small comments.

File: Mathlib/Order/SymmDiff.lean (hunk 1 above)
Comment (EtienneC30) [reply Mathlib/Order/SymmDiff.lean]: Thanks for this! I did not know about `convert`.

**Comment by fpvandoorn** [comment]
Thanks!

bors d+

You can write `bors merge` once all tests have passed (the latest commit shows a green checkmark).

**Comment by EtienneC30** [comment]
bors merge

**Pull request closed without merging**